DllImportGenerator/DllImportGenerator/DllImportGenerator.cs

```diff
@@ -112,8 +130,11 @@ public void Execute(GeneratorExecutionContext context)
                     generatorDiagnostics.ReportConfigurationNotSupported(lcidConversionAttr, nameof(TypeNames.LCIDConversionAttribute));
                 }
 
+                // TODO: These need to be propagaged to the P/Invoke target, which is this does not do correctly.
```

Comment:
Why don't these get propagated properly? Spelling and grammar check please.

Reply:
This is an old todo that I need to remove (I've fixed this)